src/main/java/org/broadinstitute/hellbender/tools/exome/allelefraction/AllelicFractionGlobalParameters.java

```diff
@@ -6,12 +6,12 @@
  *
  * @author David Benjamin &lt;davidben@broadinstitute.org&gt;
  */
-public class AllelicBiasParameters {
+public final class AllelicFractionGlobalParameters {
```

Comment:
any reason for choosing `AllelicFractionGlobalParameters` over `AlleleFractionGlobalParameters`?